sdk/python/kfp/v2/dsl/artifact.py

```diff
@@ -148,15 +155,18 @@ def __init__(self, instance_schema: Optional[str] = None):
     # Initialization flag to prevent recursive getattr / setattr errors.
     self._initialized = True
 
-  def _get_artifact_type(self) -> str:
+  @classmethod
```

Comment:
I feel like it's more natural in other module to write 
```
Model.get_artifact_type()
```
instead of
```
Model().get_artifact_type()
```
because the type schema should be a class attribute instead of an object attribute. And also it does not require instantiating an object each time, which gives it a slight margin in the efficiency perspective. WDYT?

Reply:
Makes sense. Thanks!